src/lib/utils/synapseTypes/Entity/Entity.ts

```diff
@@ -56,9 +56,13 @@ export interface Versionable {
   versionNumber?: number
 }
 
-type VERSIONABLE_ENTITY_CONCRETE_TYPE =
-  | FILE_ENTITY_CONCRETE_TYPE
-  | TABLE_CONCRETE_TYPE
+export const VERSIONABLE_ENTITY_CONCRETE_TYPE_VALUES = [
+  FILE_ENTITY_CONCRETE_TYPE_VALUE,
+  ...TABLE_CONCRETE_TYPE_VALUES,
+] as const
```

Comment:
`as const` ensures that we get the specific string values when we call `typeof VALUES[number]`